src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -79,9 +79,11 @@ def set_top_tags(
         )
         scope.set_tag("stacktrace_link.platform", ctx["platform"])
         scope.set_tag("stacktrace_link.code_mappings", has_code_mappings)
+        scope.set_tag("stacktrace_link.file", ctx["file"])
+        scope.set_tag("stacktrace_link.abs_path", ctx["abs_path"])
         if ctx["platform"] == "python":
             # This allows detecting a file that belongs to Python's 3rd party modules
-            scope.set_tag("stacktrace_link.in_app", "site-packages" not in str(ctx["file"]))
+            scope.set_tag("stacktrace_link.in_app", "site-packages" not in str(ctx["abs_path"]))
```

Comment:
We already have this in the context.
https://github.com/getsentry/sentry/blob/cf80efc55fa8f79e5ff9106494f15c7248cf8769/src/sentry/api/endpoints/project_stacktrace_link.py#L62